README.md

```diff
@@ -57,15 +64,26 @@ There some predefined error scenarios can be triggered, like connection issues.
 
 In order to extend the available cases take a look at [scenarios.go](./broken/scenario.go).
 
+Once started, use the web interface to inject an appropriate error. If repeated errors (of the same kind) are wanted, start as many instances of the web interface as wanted and inject the fault once per instance.
```

Comment:
Maybe change "instances" to "tabs", otherwise it sounds like multiple broken instances to me

Reply:
This is perhaps something we should improve in the UI instead btw? 😄

Reply:
Done